scss/dpt_classic.scss

```diff
@@ -564,6 +564,8 @@ div#dialogFrame {
   width: 400px;
   left: 50%;
   padding: 10px;
+  display: flex;
+  flex-direction: column;
```

Comment:
Great. 
Now we need to apply this change to every theme, not just to the classic one.
I think that the cleanest way to do so is to move the `#form` layout related properties to `layout.scss`, so that only theme related properties (border, coloring etc.) will remain in `dpt_*.scss`.